$OpenBSD: patch-meson_build,v 1.2 2022/02/15 10:30:23 ajacoutot Exp $

Fix undefined reference errors to _Unwind_* on clang+ld.bfd archs;
it may cause runtime issues on lld archs as well.

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -223,7 +223,6 @@ libosinfo_check_cflags = [
     '-Wframe-larger-than=4096',
     '-Wno-overlength-strings',
     '--param=ssp-buffer-size=4',
-    '-fexceptions',
     '-fasynchronous-unwind-tables',
     '-fdiagnostics-show-option',
     '-funit-at-a-time',
